Tests/RingPublishingTrackingTests/DecoratorTests.swift

```diff
@@ -177,6 +177,7 @@ class DecoratorTests: XCTestCase {
         let params = decorator.parameters
 
         XCTAssertNotNil(params["RDLU"], "RDLU should not be empty")
+        XCTAssertNotNil(params["IZ"], "IZ should not be empty")
```

Comment:
w sumie to można by było więcej oczekiwać od testu takiego.. np. sprawdzenia poprawnej struktury parametru ;)